native/cocos/ui/webview/WebViewImpl-openharmony.cpp

```diff
@@ -150,12 +152,18 @@ void WebViewImpl::reload() {
 
 bool WebViewImpl::canGoBack() {
     // TODO(qgh):OpenHarmony does not support this interface.
-    return true;
+    //bool isCanGoBack = false;
+    //NapiHelper::postSyncMessageToUIThread("canGoBack", _viewTag, &isCanGoBack);
+    //return isCanGoBack;
+	return true;
```

Comment:
Debug codes?

Reply:
This is native implementation sends synchronous messages to worker, which is currently problematic. So it is commented out.

The whole file is not used, it is left because native binds the video related interface. It may need to be handled separately.